dwitter/templates/feed/feed.html

```diff
@@ -59,9 +60,17 @@
   {% include 'snippets/new_dweet_card.html' %}
   {% endif %}
 
-  {% for dweet in dweet_list %}
-    {% include 'snippets/dweet_card.html' %}
-  {% endfor %}
+  {% if request.user.is_authenticated %}
+      {% for dweet in dweet_list %}
+        {% include 'snippets/dweet_card.html' %}
+      {% endfor %}
+  {% else %}
+      {% for dweet in dweet_list %}
+        {% cache 900 dweetcard dweet.pk %}
```

Comment:
Is 900 seconds (15 min) cache on each individual dweet a bit too much? Granted, it's just for people who haven't logged in, and it will still display new dweets on time. It should only affect Awesome-count and comments.
Works well when we're under heavy load, but for normal operations we might want to turn it down?

But to how much?